modules/README.md

```diff
@@ -244,7 +264,7 @@ interface ISafeProtocolSignatureValidator {
     /**
      * @param safe The Safe that has delegated the signature verification
      * @param sender The address that originally called the Safe's `isValidSignature` method
-     * @param hash The EIP-712 hash whose signature will be verified
+     * @param messageHash The EIP-712 hash whose signature will be verified
```

Comment:
`messageHash` is now used for different purposes in this PR.

In the example on the routing `messageHash` refers to the result of `hashStruct` of the EIP-712 spec (we could rename this to `structHash`).

And here it refers to the final EIP-712 hash.

Reply:
done